TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/Box.kt

```diff
@@ -0,0 +1,48 @@
+package org.firstinspires.ftc.teamcode.api
+
+import com.qualcomm.robotcore.eventloop.opmode.OpMode
+import com.qualcomm.robotcore.hardware.Servo
+
+object Box : API() {
```

Comment:
Could you please add documentation giving a general overview of what this API does? It doesn't have to be long. You can look at other API documentation for reference.

P.S. use `/** */` for doc comments instead of `/* */` :)

Reply:
done :)